crates/cargo-util-schemas/src/manifest.rs

```diff
@@ -10,15 +10,16 @@ use std::fmt::{self, Display, Write};
 use std::path::PathBuf;
 use std::str;
 
-use anyhow::Result;
 use serde::de::{self, IntoDeserializer as _, Unexpected};
 use serde::ser;
 use serde::{Deserialize, Serialize};
 use serde_untagged::UntaggedEnumVisitor;
 
 use crate::core::PackageIdSpec;
 use crate::core::PartialVersion;
+use crate::core::PartialVersionError;
 use crate::restricted_names;
+pub use crate::restricted_names::NameValidationError;
```

Comment:
nit: can you separate this `use` with a newline so that it being part of the public API stands out more?